ggshield/scan/scanner.py

```diff
@@ -228,7 +230,7 @@ def _scan_chunk(
             self.client.multi_content_scan,
             documents,
             self.headers,
-            ignore_known_secrets=self.ignore_known_secrets,
+            ignore_known_secrets=True,
```

Comment:
If I am not mistaken, on the server side this argument is now ignored, is this correct? If so, I would rather remove it from here than pass an ignored value.

Reply:
it's not ignored. Finally we want the information about incident url to be returned only for ggshield, so using this argument will say gim to include this info